Add documentation page for duplicate_os_installation unhealthy state (#40022)
home-assistant/home-assistant.github.io: source/more-info/unhealthy/duplicate_os_installation.md

---
title: "Duplicate Home Assistant OS installation"
description: "How to resolve issues caused by having multiple Home Assistant OS installations on the same system."
---

## The issue

> Multiple Home Assistant OS installations have been detected, which can cause serious system stability issues. This problem typically occurs when you have installed Home Assistant OS multiple times on different storage devices (like SD cards, USB drives, or internal storage) that are all connected to the same device. This mismatch can lead to:

Reviewer: "stability issues" sounds like a runtime issue, maybe something like this?

```suggestion
Multiple Home Assistant OS installations have been detected, which can lead to non-bootable systems, especially after updates. This problem typically occurs when you have installed Home Assistant OS multiple times on different storage devices (like SD cards, USB drives, or internal storage) that are all connected to the same device. This mismatch can lead to:
```

Author: And is it sure the disk reference by label is stable when multiple devices with the same label exist? Because otherwise you can run into issues between reboots as well. But I guess that fits into your wording as well.

- Your system booting into an old version of Home Assistant OS even after updating.
- Complete boot failure, leaving your system unable to start.
- Other forms of unpredictable behavior.

## The solution

> The solution is to ensure you have only one Home Assistant OS installation on your system. To do that, follow these steps:

Reviewer: ```suggestion
Make sure you have only one Home Assistant OS installation on your system. To do that, follow these steps:
```
